src/aiven_gatekeeper.c

```diff
@@ -36,6 +36,13 @@
 
 PG_MODULE_MAGIC;
 
+/* session_auth_is_superuser was renamed to current_role_is_superuser */
+#if PG_VERSION_NUM >= 170000
```

Comment:
Can we add a constant like in https://github.com/Aiven-Open/aiven-pg-security/blob/1da6e180178809e31c238d20181a8a09b0d58943/src/aiven_gatekeeper.h#L16 and use that?